libs/features/mas/src/sidenav/merch-sidenav-checkbox-group.js

```diff
@@ -53,7 +53,7 @@ export class MerchSidenavCheckboxGroup extends LitElement {
         this.prepend(h3El);
 
         [...this.children].forEach((el) => {
-          if (el.id && el.id !== id) {
+          if (!el.id) {
```

Comment:
I think, we need to group them together rather than having a separate group per checkbox

![image](https://github.com/user-attachments/assets/4d7750b4-e5b5-443b-bdb3-329d64f1eb69)

From GPT  :)
```
From an accessibility (a11y) standpoint, a single checkbox should not have role="group". Here's why:
role="group" is used to group multiple related elements together under a shared label (for example, a group of checkboxes for "Preferred Contact Methods": Email, Phone, SMS).
```

Reply:
You are absolutely right. And that's also how they described it but I was blind.
```
<p id="contact-label">Preferred contact method</p>
<div role="group" aria-labelledby="contact-label">
  <label for="email">Email</label><input type="radio" id="email" name="contact">
  <label for="phone">Phone</label><input type="radio" id="phone" name="contact">
</div>
```
Fixed. We already have a group so I added `role="group" aria-labelledby="sidenav-checkbox-group-title"` only there in the root element.